### DISCUSSION TITLE
[SUGGESTION] Small boiler

### DISCUSSION THREAD

**Discussion by PanmanS** [PeasantFoundry/Mechtrauma#47] [Ideas]
I think it would be nice to have some small variant of boiler, similiar to like you did with reduction gear, to allow players to have smaller subs fully mechtraumized without having Goofy Ahh Boiler that is `0.125` scale, just to fit it inside!

**Comment by PeasantFoundry** [comment]
Well, there is this alternate skin prototype for a horizontal steam boiler/generator. 
![image](https://user-images.githubusercontent.com/113934941/210440207-c3214918-6a4e-48ca-b726-6102c2f01af7.png)

**Comment by PanmanS** [comment]
that would be amazing as it will finally help me to mechtraumize all subs!
